lib/workers/branch/npmrc.js

```diff
@@ -0,0 +1,17 @@
+const ini = require('ini');
+
+module.exports = { validateNpmrc };
+
+function validateNpmrc(input) {
```

Comment:
If this function is used nowhere else, it can be  moved into `lib/config/validation.js`